### DISCUSSION TITLE
عدم نمایش حجم مصرفی و تعداد روزهای باقی مانده کاربر

### DISCUSSION THREAD

**Discussion by rjmomo** [hiddify/Hiddify-Manager#426] [Q&A سوالات رایج]
با سلام و تشکر از زحمات شما
چرا مقدار حجم مصرفی کاربر و تعداد روز های باقی مانده کاربر را نشان داده نمی شود؟

**Comment by zZedix** [comment]
متاسفانه اون امکان به دلیل شکایت یه سری از دوستان فعلا حذف شده ! واقعا امکان عالی بود

**Reply by rjmomo** [reply]
اگر می شد بصورت ماژول باشه که هرکسی خواست فعالش کنه ،خوب می شد
